src/lib/fixtures.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+const models = require('../models')
+
+function Fixtures (db, config) {
```

Comment:
I think this is a good example of when OOP causes design problems. The function `setupHoldAccount` has access to the `config` object even though it doesn't need it, which means we have introduced unnecessary coupling. Also `db` is never used in this class.

I think these should just be two regular functions without any class. Then the linter should warn you that you have unused arguments.